services-examples-parent/services-examples-runner/src/main/java/io/scalecube/services/examples/ExamplesRunner.java

```diff
@@ -81,8 +81,8 @@ private static ServiceDiscovery serviceDiscovery(ServiceEndpoint serviceEndpoint
     return new ScalecubeServiceDiscovery(serviceEndpoint)
         .options(
             opts ->
-                opts.seedMembers(config.seedAddresses())
-                    .port(config.discoveryPort())
+                opts.membership(cfg1 -> cfg1.seedMembers(config.seedAddresses()))
```

Comment:
this api change is less clean... less expect-able.

Reply:
With this api change cluster configuration comes in groups: MembershipConfig, TransportConifg, GossipConfig and ClusterConfig  itself. It was very inconvenient when settings from all sub-components were living in global CluserConfig. Now with separation of configuration options it's clean what properties has gossip, what properties  has FailureDetector, and so on.